VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -98,6 +104,7 @@ export class VideoCallService {
      * @param maxBandwidth the maximum bandwith
      */
     makeCall(pexipNode: string, conferenceAlias: string, participantDisplayName: string, maxBandwidth: number) {
+        this.initCallTag();
```

Comment:
does setting in make call meet this AC? 
"value should be unique every time the user disconnects and re-connects to pexip"

Reply:
This has to be set before make call. In the components, we call makeCall on unexpected disconnects